site-src/blog/2023/0829-mesh-support.md

```diff
@@ -0,0 +1,210 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where service
+  mesh support has has now reached Experimental status, we've introduced CEL
+  validation and a Mesh conformance profile, and more!
+---
+
+# Gateway API v0.8.0: Introducing Service Mesh Support!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 29, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of Gateway API! With this
+release, Gateway API support for service mesh has reached [Experimental
+status][status], we've introduced CEL validation and a `Mesh` conformance
+profile, and we've made some changes to the GEP process. We look forward to
```

Comment:
I don't think we need to mention the other items here. I'd be on board with removing them from the intro altogether, or just minimizing them into something like this:
```suggestion
status][status], we've also introduced some smaller changes like CEL validation. We look forward to
```

Reply:
I took a (slightly tweaked) version of this; I think the point is sound.